src/main/java/net/sf/jabref/gui/actions/version/VisitDownloadPageAction.java

```diff
@@ -0,0 +1,37 @@
+/*  Copyright (C) 2016 JabRef contributors.
+    This program is free software; you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation; either version 2 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License along
+    with this program; if not, write to the Free Software Foundation, Inc.,
+    51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+*/
+package net.sf.jabref.gui.actions.version;
+
+import java.awt.event.ActionEvent;
+
+import javax.swing.AbstractAction;
+
+import net.sf.jabref.gui.desktop.JabRefDesktop;
+import net.sf.jabref.logic.l10n.Localization;
+import net.sf.jabref.logic.util.version.VersionChecker;
+
+public class VisitDownloadPageAction extends AbstractAction {
+
+    public VisitDownloadPageAction() {
+        super(Localization.lang("Download_update"));
+    }
+
+    @Override
+    public void actionPerformed(ActionEvent e) {
+        JabRefDesktop.openBrowserCatchException(VersionChecker.JABREF_DOWNLOAD_URL);
```

Comment:
In my opinion you should create a new method `VersionChecker.goToLatestVersionDownload()` and put the `JabrefDesktop.openBrowserCatchException` method in there. This way `JABREF_DOWNLOAD_URL` can be private.